website/docs/r/kms_grant.html.markdown

```diff
@@ -74,3 +74,11 @@ In addition to all arguments above, the following attributes are exported:
 
 * `grant_id` - The unique identifier for the grant.
 * `grant_token` - The grant token for the created grant. For more information, see [Grant Tokens](http://docs.aws.amazon.com/kms/latest/developerguide/concepts.html#grant_token).
+
+## Import
+
+KMS Grants can be imported using the `id`, e.g.
+
+```
+$ terraform import aws_kms_grant.test 1234abcd-12ab-34cd-56ef-1234567890ab
+```
```

Comment:
It appears the import ID for this should include the grant ID as well 👍 

``````suggestion
KMS Grants can be imported using the Key ID and Grant ID separated by a colon (`:`), e.g.

```
$ terraform import aws_kms_grant.test 1234abcd-12ab-34cd-56ef-1234567890ab: abcde1237f76e4ba7987489ac329fbfba6ad343d6f7075dbd1ef191f0120514
```
``````